### ISSUE TITLE
Disabled Button is not clickable after being enabled

### ISSUE THREAD

**Issue by StorytellerCZ** [reakit/reakit#480]
### Current behavior

When having a Button with `type="submit"` in a form renders the form unsubmitable. Can't trigger the submit action from input fields nor clicking at the submit button.

### Steps to reproduce the bug
1. Open sandbox: https://codesandbox.io/s/stoic-poitras-1f4or
2. Write something and try to click the button (or hit enter in the input field), nothing happens.
3. Change Reakit button to standard button in the code.
4. Click the button and you will see your input in the console.
5. Change the version of Reakit version to 1.0.0-beta.9 and use Reakit Button again and the form will work correctly.

### Expected behavior

Button with input `type="submit"` should submit the form it is part of.

### Possible solutions

Not really sure. Just noticed this today (2019-11-07) after upgrading to beta.10 (from beta.9). The only suspicious thing I can find in the changelog is #474, but that looks pretty innocent to me.

### Environment

```sh
$ npx envinfo --system --binaries --browsers --npmPackages "{react*,reakit*}"

System:
    OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
    CPU: (4) x64 Intel(R) Core(TM) i7-6500U CPU @ 2.50GHz
    Memory: 4.49 GB / 15.55 GB
    Container: Yes
    Shell: 5.0.3 - /bin/bash
  Binaries:
    Node: 12.11.1 - /usr/bin/node
    Yarn: 1.19.1 - /usr/bin/yarn
    npm: 6.12.1 - /usr/bin/npm
  Browsers:
    Chrome: 73.0.3683.103
    Firefox: 70.0.1
  npmPackages:
    react: ^16.11.0 => 16.11.0 
    react-accessible-accordion: ^3.0.0 => 3.0.0 
    react-apollo: ^3.1.3 => 3.1.3 
    react-dom: ^16.11.0 => 16.11.0 
    react-final-form: ^6.3.0 => 6.3.0 
    react-final-form-arrays: ^3.1.1 => 3.1.1 
    react-final-form-listeners: ^1.0.2 => 1.0.2 
    react-helmet: ^5.2.1 => 5.2.1 
    react-intl: ^3.4.0 => 3.4.0 
    react-loading: ^2.0.3 => 2.0.3 
    react-portal: ^4.2.0 => 4.2.0 
    react-router-dom: ^5.1.2 => 5.1.2 
    react-select: ^3.0.8 => 3.0.8 
    reakit: ^1.0.0-beta.10 => 1.0.0-beta.10 
```

**Comment by tom-sherman** [comment]
This does seem to be an issue with Reakit's `Button`. The following code (using a  `<button>`) triggers the onsubmit callback.

Codesandbox: https://codesandbox.io/s/festive-rain-soyzl

```javascript
import React from "react";
import ReactDOM from "react-dom";
import { Form, Field } from "react-final-form";
import { Button } from "reakit";

import "./styles.css";

const submit = (values, form, complete) => {
  console.log(values);
  complete();
  form.reset();
};

function App() {
  return (
    <div className="App">
      <Form
        onSubmit={submit}
        subscription={{ submitting: true, pristine: true, submitError: true }}
        render={({ handleSubmit, submitting, pristine }) => (
          <form onSubmit={handleSubmit}>
            <Field name="text" type="text" component="input" />
            <button type="submit">Submit</button>
          </form>
        )}
      />
    </div>
  );
}

const rootElement = document.getElementById("root");
ReactDOM.render(<App />, rootElement);
```

**Comment by tom-sherman** [comment]
This seems to be an issue with using the render prop of `react-final-form` in combination with a Reakit submit button. When using a native form element, Reakit beta 10 works as expected: https://codesandbox.io/s/withered-dust-uf08c

**Comment by diegohaz** [comment]
It's indeed a bug. Working on a fix.

**Comment by StorytellerCZ** [comment]
Thanks for the fix! I found it at 1 in the morning so sorry for the report not being a better quality.
Any chance that this could be released soon?

**Comment by diegohaz** [comment]
No worries! Thanks for reporting it! I'll cut a release today or tomorrow at the most.
By the way, thanks for the donation! ❤️

**Comment by StorytellerCZ** [comment]
Thanks! I'm in a pitch, because if I go back to beta.9 I run into another error in my app that has been fixed in beta.10. :sweat_smile: 
I wish I could donate more and regularly, but sadly I have been burning through my savings for a while.

**Comment by diegohaz** [comment]
It's been released as `v1.0.0-beta.11`!
